portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,10 @@ function addMusic() {
   const musicContainer = document.getElementById('music-container');
   musicContainer.innerText = musician;
 }
+
+async function showMessage() {
```

Comment:
Why didn't you indent this? Please follow the Google coding style for your JavaScript per my message to the team chat.